library/ssl_tls.c

```diff
@@ -648,7 +648,7 @@ static int tls_prf_generic( mbedtls_md_type_t md_type,
     psa_set_key_algorithm( &key_attributes, alg );
     psa_set_key_type( &key_attributes, PSA_KEY_TYPE_DERIVE );
 
-    status = psa_import_key( &key_attributes, secret, slen, master_slot );
+    status = psa_import_key( &key_attributes, secret, slen, &master_slot );
```

Comment:
Ah, now it's fixed. Please apply fixup commits to the original commits so we don't have to waste time reviewing and pointing out problems you've already fixed.

Reply:
I have rebased the PR to apply the fixes.

Reply:
Thanks